components/button/themes/spectrum-two.css

```diff
@@ -138,10 +138,10 @@
 			--spectrum-button-border-color-down: transparent;
 			--spectrum-button-border-color-focus: transparent;
 
-			--spectrum-button-content-color-default: var(--spectrum-white);
-			--spectrum-button-content-color-hover: var(--spectrum-white);
-			--spectrum-button-content-color-down: var(--spectrum-white);
-			--spectrum-button-content-color-focus: var(--spectrum-white);
+			--spectrum-button-content-color-default: var(--spectrum-gray-25);
+			--spectrum-button-content-color-hover: var(--spectrum-gray-25);
+			--spectrum-button-content-color-down: var(--spectrum-gray-25);
+			--spectrum-button-content-color-focus: var(--spectrum-gray-25);
```

Comment:
I've only changed the S2 mappings for the `button-content-color` as having `--spectrum-white` was resulting in a inaccessible state for the spectrum-two primary button in dark mode. Let me know if it works now? @jawinn

Reply:
Like Rise mentioned, we'd need to set these to `--spectrum-white` in the Spectrum 1/Legacy `spectrum.css` to maintain existing S1 values, since the value is changing for Spectrum 2.